src/custom-element.js

```diff
@@ -707,6 +704,7 @@ function createBaseCustomElementClass(win) {
         this.implementation_.layoutWidth_ = this.layoutWidth_;
       }
       // TODO(malteubl): Forward for stubbed elements.
+      // TODO(jridgewell): We should pass the layoutBox down.
```

Comment:
Ok, that'd be fine, of course. But we'd keep `getLayoutBox()` method itself? I made more notes on this below as well. Ideally this would be only size. However, in the meantime, what this API promises is to return "the last measured/known layout box".